modules/cco-ccoctl-configuring.adoc

```diff
@@ -37,6 +37,48 @@ endif::alibabacloud[]
 The `ccoctl` is a Linux binary that must run in a Linux environment.
 ====
 
+.Prerequisites
+
+ifdef::aws-sts[]
+* You have created an AWS account for the `ccoctl` to use with the following permissions:
+
```

Comment:
Need a "+" here so that the following table is indented properly with the above bullet